__tests__/create.util.ts

```diff
@@ -1,26 +1,37 @@
-import Bull = require('bull');
+import { Processor, Queue, QueueEvents, Worker } from 'bullmq';
+import RedisClient from 'ioredis';
 import { Registry } from 'prom-client';
 
 import { makeGuages, QueueGauges } from '../src/queueGauges';
 
 export interface TestData {
-  name: string;
-  queue: Bull.Queue;
-  prefix: string;
-  guages: QueueGauges;
-  registry: Registry;
+	name: string;
+	queue: Queue;
```

Comment:
Unfortunately there seems to be a bunch of formatting changes from the base repo, as I applied our own prettier config.
I advise to flip the _Hide whitespace_ option in the settings cog above, to the right of _Conversations_.